lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -38,7 +38,7 @@ const PolygonParameterEditor = React.createClass({
     },
 
     getValue() {
-        const pointsLongLats = this.props.parameterValues[this.props.parameter.id];
+        const pointsLongLats = this.props.parameterValues[this.props.parameter.id][0];
```

Comment:
Although I realise this isn't caused at all by your PR - sorry!  Perhaps I should really open this as a new issue.